microprofile/lra/participant/src/main/resources/schema/wildfly-microprofile-lra-participant_1_0.xsd

```diff
@@ -23,7 +23,7 @@
           ]]>
         </xs:documentation>
       </xs:annotation>
-      <xs:attribute name="lra-coordinator-url" type="xs:string" default="http://localhost:8080/lra-coordinator/lra-coordinator"/>
+      <xs:attribute name="lra-coordinator-url" type="xs:string" default="http://localhost:8080/lra-coordinator"/>
```

Comment:
To be truly correct there needs to be a new 2.0 version of this schema, with 1.0 unchanged.  We'd need to tweak MicroProfileLRAParticipantSubsystemSchema.getXMLDescription() as well so when it processes the 1.0 namespace and sees no setting for this attribute it explicitly writes the old value.

The problem is AIUI we don't actually want to do that 'correct' behavior because it won't work -- the local coordinator won't understand that old URL.

Probably there should be a 2.0 schema, and the 1.0 documentation text of the 1.0 schema can be updated to explain the behavior on newer servers that support 2.0? Or just leave it unexplained? The thing is the 1.0 xsd exists and is published and still has meaning in contexts other than a WF 35+ server.

Reply:
I could add `Please note that the default value will no longer work to connect to WildFly servers incorporating WFLY-20050` to https://github.com/wildfly/wildfly/blob/main/microprofile/lra/participant/src/main/resources/schema/wildfly-microprofile-lra-participant_1_0.xsd#L20 (I would probably change `The "url"` to `The "lra-coordinator-url"` too) as a separate new WFLY (maybe a Bug or a Task "Update text in schema wildfly-microprofile-lra-participant_1_0.xsd") if you think that would be helpful (and also importantly, if that is the kind of wording change you would agree is correct :) )? Possibly we can do that as a follow up after this pull request?

Reply:
We can definitely do that as a follow up.

The wording is tricky as what will happens depends on the context when a document using this 1.0 schema is parsed. What you wrote is correct if the context is a server before WF 35. If the context is WF 35 or later reading an old config that has a 1.0 schema element, then it won't actually use the default stated in the XSD; it will use the new default.

But anyway, NBD and can definitely be dealt with later.